src/ert/scheduler/job.py

```diff
@@ -63,13 +63,20 @@ def __init__(self, scheduler: Scheduler, real: Realization) -> None:
         self.state = JobState.WAITING
         self.started = asyncio.Event()
         self.returncode: asyncio.Future[int] = asyncio.Future()
-        self._aborted = False
         self._scheduler: Scheduler = scheduler
         self._callback_status_msg: str = ""
         self._requested_max_submit: Optional[int] = None
         self._start_time: Optional[float] = None
         self._end_time: Optional[float] = None
 
+    def unschedule(self, msg: str) -> None:
+        self.state = JobState.ABORTED
```

Comment:
Send some information to the logger as well in this scenario?

Reply:
Is it possible to test if/how this information is conveyed to the GUI user?

Reply:
![image](https://github.com/user-attachments/assets/5cf71627-e765-417a-893e-bc2b0082fa8b)

Reply:
Unfortunately just shows as waiting: 
![image](https://github.com/user-attachments/assets/a59d2dae-c77d-4b26-bca4-1c1fe47ac1de)
We should probably fix that first. Note that this scenario also makes runpath deletion in the GUI wonky.

Reply:
Latest changes marks the realization as failed correctly: 
![image](https://github.com/user-attachments/assets/f4cd8887-88e2-4ce7-acf6-a5ad7c7fb62f)

Reply:
But there is no message.

Reply:
Now shows with hover 
[Screencast from 2024-10-09 08-20-19.webm](https://github.com/user-attachments/assets/4ae1d4a8-acc3-4188-8244-b6515282d700)

Reply:
Also added a log message